src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte

```diff
@@ -110,10 +110,16 @@
 				destination,
 				amount,
 				utxosFee,
-				progress,
 				network,
 				source,
-				identity: $authIdentity
+				identity: $authIdentity,
+				onProgress: () => {
+					if (sendProgressStep === ProgressStepsSendBtc.INITIALIZATION) {
+						progress(ProgressStepsSendBtc.SEND);
+					} else if (sendProgressStep === ProgressStepsSendBtc.SEND) {
```

Comment:
just a smaller question, unrelated: did the linter flagged something here or not?

Reply:
Nope, linter didn't complain here. I think it would do it if I had something like:
```
if (sendProgressStep === ProgressStepsSendBtc.INITIALIZATION) {
  progress(ProgressStepsSendBtc.SEND);
  return;
} else if (sendProgressStep === ProgressStepsSendBtc.SEND) {
...
```